src/version-bump.ts

```diff
@@ -77,9 +77,11 @@ export async function versionBump(arg: (VersionBumpOptions) | string = {}): Prom
     }
     else {
       console.log(symbols.info, 'Executing script', operation.options.execute)
-      await x(operation.options.execute, [], {
+      const [command, ...args] = operation.options.execute.split(' ')
```

Comment:
Well, this is honestly tricky as this would usually need a proper parser to handle cases like `-m "some space in side"`. It's a shame the tinyexec doesn't support the command mode.

Reply:
[execa](https://www.npmjs.com/package/execa) supports the command mode.
Are there any other recommended libraries?

Reply:
I brought it upstream: https://github.com/tinylibs/tinyexec/issues/42